qa/rpc-tests/fundrawtransaction.py

```diff
@@ -678,14 +678,18 @@ def run_test(self):
         assert(signedtx["complete"])
         self.nodes[0].sendrawtransaction(signedtx["hex"])
 
+        self.sync_all()
```

Comment:
I guess there are many places where we do this.
What about fixing all together in an upcoming PR?